src/interface/web/app/components/chatMessage/chatMessage.tsx

```diff
@@ -180,6 +181,10 @@ export interface StreamMessage {
     inferredQueries?: string[];
     turnId?: string;
     queryFiles?: AttachedFileText[];
+    excalidrawDiagram?: string;
+    generatedFiles?: AttachedFileText[];
+    generatedImages?: string[];
+    generatedExcalidrawDiagram?: string;
```

Comment:
Ah good catch. Need to just pick one, no difference. Currently just using `excalidrawDiagram`.

Reply:
Actually, generatedExcalidrawDiagram holds the new asset that is created by Khoj and sent over stream. These are differentiated in the code for rendering purposes.

Reply:
Is the generatedExcalidrawDiagram asset a json or base64 image? And what does the `excalidrawDiagram` hold? Still unsure of the difference between what the two variables contain

Reply:
It's a JSON asset, all of the excalidraw outputs are always JSON assets. The `generated` prefix for all these fields is a differentiation between streamed data and loaded data. If streamed, it will be populated in `generatedx`, if loaded, it will be populated in the field without the prefix. That's why the `generated` fields are only on `StreamMessage`.